client_sdk/avalon_client_sdk/quorum/__init__.py

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Add Apache 2.0 license.

Reply:
Updated in the latest commit.Please review